media_test_harness/testcase.py

```diff
@@ -64,3 +55,12 @@ def run_playback(self, interval=1, set_duration=None,
                                   video, playback_done)
                 except VideoException as e:
                     raise self.failureException(e)
+
+    def skipTest(self, reason):
```

Comment:
In a test, you might make a call `self.skipTest('Skipping because this should only work with Netflix')` for example. I ran into the issue while adding skip logic to the 'dormant' test I was working on yesterday.